src/components/pages/VolunteersPage/InviteVolunteerModal/InviteVolunteerModal.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Button, Form, Modal } from 'react-bootstrap';
+
+class InviteVolunteerModal extends React.Component {
+  state = {
```

Comment:
You should decide whether you want to use controlled or uncontrolled components. Right now, you are updating state and not really using it for anything.

https://reactjs.org/docs/forms.html#controlled-components
https://reactjs.org/docs/uncontrolled-components.html

Reply:
I see

Reply:
I have made it uncontrolled